src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -128,8 +128,26 @@ def truncate_text(str_to_shorten, width=70, placeholder=' [...]'):
 
 
 def get_installed_cli_distributions():
-    from pkg_resources import working_set
-    return [d for d in list(working_set) if d.key == CLI_PACKAGE_NAME or d.key.startswith(COMPONENT_PREFIX)]
+    # Stop importing pkg_resources, because importing it is slow (~200ms).
+    # from pkg_resources import working_set
+    # return [d for d in list(working_set) if d.key == CLI_PACKAGE_NAME or d.key.startswith(COMPONENT_PREFIX)]
+
+    # Use the hard-coded version instead of querying all modules under site-packages.
+    from azure.cli.__main__ import __version__ as azure_cli_version
```

Comment:
From `azure-cli-core` perspective, we should not assume there is a `azure-cli` repo. That is to say, `azure.cli.__main__` may not exist.

Reply:
Good catch! As now `azure-cli` and `azure-cli-core` have the same version, we don't need this dependency anymore. 

Also importing `azure.cli.__main__` will cause `__main__` being executed, thus causing CI failure:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=508652&view=logs&jobId=4ddf4a4c-924b-5e90-5386-b844cc2d2073&j=4ddf4a4c-924b-5e90-5386-b844cc2d2073&t=d1586226-7623-5d9f-0d54-f083e3aeff70

``` 
=====================
| Discovering Tests |
=====================

az: 'test' is not in the 'az' command group. See 'az --help'. If the command is from an extension, please make sure the corresponding extension is installed. To learn more about extensions, please visit https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview

The most similar choice to 'test' is:
	rest
```